cpufreq additions

xdel opened the pull request.

Commit:
Ceilometer integration

  * BP: ceilometer-fuel-integraition
  * HA and simple deployment
  * general MySQL backend for ceilometer
  * tested on CentOS 6.4
  * need verification of Ubuntu support

aglarendil committed:
[PRD-1983] tie services' config to AMQP Server

Commit:
Fixup some un-needed string interpolations in horizon

mattymo committed:
add ntp to nailgun

add resolv.conf and dhcp-enter-hook for nailgun

make ntp template for fuel master server

add ntp config template to openstack::clocksync

make ntp template for fuel master server

Commit:
[PRD-2360] Radosgw wont start on multiple controllers

refactor radosgw key creation so that its idempotent.

Commit:
always enable haproxy for swift or radosgw in HA

Setup Swift HAProxy when Swift is enabled either for glance or for
objects, not just when it's enabled for glance. Setup RadosGW HAProxy
when Swift is disabled and RadosGW is enabled. Use active-passive
balancing for RadosGW.

mattymo committed:
add ipsort function for RHOS deployment order

bogdando committed:
Define CPU mode none for virtual, host-model for HW cases

Fixes: #PRD-1950

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

Commit:
Add support for older cciss devices

add detection for if devname is cciss and insert seperator if necessary

Commit:
Add cciss device support for Ubuntu

bogdando committed:
Fix isvirtual flag for logging class

Use fact is_virtual to determine the $virtual value

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix isvirtual case only for checksum_bootpc as well

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

Commit:
Find devices by partition GUID and exclude mounted devices

Facter function now uses partition GUID instead of filesystem label
to find matching devices. It also excludes already mounted devices.
This depends on https://review.openstack.org/#/c/54841/.
See also https://bugs.launchpad.net/fuel/+bug/1246513

Commit:
cpufreq tuning for compute nodes

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:-1: Test FAILed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_syntax_check/464/

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_pullrequest_systest/31/

Commit:
Add API reconnect retry support

Commit:
pseudo package for Centos w/ 4.0

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:-1: Test FAILed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_syntax_check/487/

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

xdel commented:
Depends on https://review.openstack.org/#/c/55526/

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:-1: Test FAILed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_pullrequest_systest/50/

Commit:
disable name resolving for ping command

aglarendil committed:
[PRD-2463] add debug string with node role

Commit:
Enable carrier waiting after interface up

Commit:
More powerfull error messages

mirantis-jenkins commented:
Merged build triggered.

Zipfer committed:
Fix swift output parse regexp

 - Bug #1248651
 - fix parse output with new swift havana release
 - fix optional whitespace at the end of the line

Zipfer committed:
specs for new swift version

Commit:
Merge pull request #824 from xenolog/wild_ip_fix

FIX: Wild IP bug.

sirmax123 committed:
Update savanna api version for havanna

sirmax123 committed:
Update heat for havanna

sirmax123 committed:
Update murano for havanna

Commit:
Fix ceph, radosgw init issues

This patch forces $::operatingsystem == Ubuntu to use the debian init script
 instead of the upstart script for radosgw. This is necessary because the
 upstart script requires a parameter that breaks puppet's upstart provider.
 this resolves #1239348.

This patch more strickly sets the dependancy for service ceph in init.pp.
 This some errors found in #1250122.

Closes-bug #1250122
Closes-bug #1249348

aglarendil committed:
Merge pull request #733 from mattymo/nailgun-ntp

Nailgun NTP improvements

Commit:
add $::ceph::use_ssl to havana

aglarendil committed:
Merge pull request #844 from Mirantis/pick_missed_fixes_to_havana

Pick missed fixes to havana

Commit:
Fix syntax errors leftover from rebase

deployment/puppet/neutron/manifests/keystone/auth.pp
 add missing $ on
 fix double quote
 fix ensure not first

deployment/puppet/openstack/manifests/nova/controller.pp
 add missing $ on 34

aglarendil committed:
Merge pull request #851 from Mirantis/havana-mopup

Havana mopup

Commit:
Fix for rabbit_hosts flag in ceilometer

aglarendil committed:
Rename quantum->neutron config resource

Closes-bug: bug1251629

aglarendil committed:
Merge pull request #852 from Mirantis/bug1251629

Rename quantum->neutron config resource

aglarendil committed:
Merge pull request #853 from Zipfer/swift_havana_fix

Swift havana fix

Commit:
Fix duplicate assignment rabbit_host

aglarendil committed:
Merge pull request #832 from xarses/bug/1249348

[4.0] Fix ceph, radosgw init issues

aglarendil committed:
Merge pull request #842 from dmitryilyin/max-update

Havanna updates for heat and murano from Max Mazur

aglarendil committed:
Merge pull request #854 from rsokolkov/master-ceilometer-havana

Master ceilometer havana

Commit:
rebased
